src/compounds/ItemPage/ItemPage.jsx

```diff
@@ -9,6 +9,7 @@ import Title from '../../components/Title/Title'
 const ItemPage = ({ descriptionStyle, img, titleStyle, ware }) => {
   const { description, id, name, snippet, slug } = ware
   // sets a default width while still allowing width to be overridden
+  img = { width: 400, ...img }
```

Comment:
this throws an error in the console for me, which is why I removed it and instead used the class img-fluid, which makes the max width the width of the column it is in.

Reply:
We might want to add a [min-width](url) to the image along with the `img-fluid` though so that very small images wont be as tiny, and large images will still be sized to the width of the column. There is no bootstrap class for min-width though so it would need to be added either in line or in a css file

Reply:
fixed the lint error by renaming the styled image. now the image has a default width (good for larger screens) and a max width (good for smaller screens).

Reply:
![image](https://user-images.githubusercontent.com/29032869/213479506-d11839a0-ed70-41c7-980c-72ff45585677.png)

Reply:
Instead of using `width: 400` can we use `min-width: 400` ? That way, images with large intrinsic sizes will be able to take up the full width of the column on larger screens instead of being shrunk to 400px, but intrinsically small images will be increased to at least 400px as they are now

for instance on a wide screen for an already large image it would be 
<img width="1582" alt="image" src="https://user-images.githubusercontent.com/73361970/213483635-6d2ef911-8460-46c6-915b-8bbbd1d61b97.png">

instead of 
<img width="1556" alt="image" src="https://user-images.githubusercontent.com/73361970/213483790-5bd1896b-17f2-4d4f-9449-db07861841c8.png">

if the image is large enough to begin with. obviously we don't want to stretch images too much, but if we are starting with already large images then i dont know if shrinking them too much is the right move

Reply:
imo it's more about the uniformity of the images. (like how they are now different sizes on the browse page). usually, images are uniformed in size on upload. if not, then in the code. we can change the `width` size to be larger than 400, but I'd still like for it to be the same across the board.

also, setting `min-width: 400` would mean that the image would no longer shrink accordingly on smaller screens.